server/services/notificationService.test.ts

```diff
@@ -0,0 +1,55 @@
+import NotificationService, { type EmailPersonalisation } from './notificationService'
+import config from '../config'
+
+const notifyApi = {
```

Comment:
Might be better to stick with notifyClient naming convention? 🤔

Reply:
Amended as suggested